ush/fv3gfs_downstream_nems.sh

```diff
@@ -135,9 +135,16 @@ for (( nset=1 ; nset <= downset ; nset++ )); do
   fi
   err_chk
 
-  # We are in a loop over downset, save output from mpmd into nset specific output
-  cat mpmd.out  # so we capture output into the main logfile
-  mv mpmd.out "mpmd_${nset}.out"
+  # We are in a loop over downset, save output from mpmd into nset
+  # specific output; this if-block is necessary for the NOAA CSP AWS
+  # platform; AWS uses `mpiexec` (PBS) as it's parallel executable
```

Comment:
WCOSS2 also uses PBS and is handled in the `elif [[ "${launcher:-}" =~ ^mpiexec.* ]]; then  # mpiexec` block.
As seen, the output of each thread is piped to `mpmd.${nm}.out`
How is AWS executing MPMD and where does its output go?  
The options above in `AWSPW.env` (`mpmd_opt="--multi-prog --output=mpmd.%j.%t.out"`) appear to be `SLURM` options.